build/Dockerfile

```diff
@@ -40,11 +40,14 @@ RUN apt-get update \
 		flex \
 		bison
 
+RUN curl -fsSL -o shfmt https://github.com/mvdan/sh/releases/download/v1.3.0/shfmt_v1.3.0_linux_amd64 && \
```

Comment:
Suggest checking this binary file is the one we expect similar to https://github.com/weaveworks/weave/blob/master/Makefile#L252